[regression] KeyError if profile does not exists

Integration tests are failing, probably because the profile is missing somewhere.

```python
                for profile in profiles:
>                   pkgs_profile = module['profiles'][profile]
E                   KeyError: 'minimal'

tests/integration/test_association.py:438: KeyError
```

--- a/tests/integration/data/associate-md.yaml
+++ b/tests/integration/data/associate-md.yaml
@@ -13,6 +13,10 @@ arches:
  - src
  - x86_64
 
+packages:
+  include: []
+  exclude: []
+
 modules:
   include:
   - name: httpd

--- a/tests/integration/data/clean-repos.yaml
+++ b/tests/integration/data/clean-repos.yaml
@@ -12,3 +12,7 @@ content_sets:
 arches:
  - src
  - x86_64
+
+packages:
+  include: []
+  exclude: []

--- a/tests/integration/test_association.py
+++ b/tests/integration/test_association.py
@@ -435,7 +435,7 @@ def test_ubipop_can_associate_modules():
             profiles = [md.profiles for md in cfg.modules.whitelist if md.name == md_name][0]
 
             for profile in profiles:
-                pkgs_profile = module['profiles'][profile]
+                pkgs_profile = module['profiles'].get(profile, [])
 
                 for pkg in pkgs_profile:
                     assert pkg in pkgs_found, '"{}" not in the repository.'.format(pkg)
@@ -449,7 +449,7 @@ def test_ubipop_can_associate_modules():
             profiles = [md.profiles for md in cfg.modules.whitelist if md.name == md_name][0]
 
             for profile in profiles:
-                pkgs_profile = module['profiles'][profile]
+                pkgs_profile = module['profiles'].get(profile, [])
 
                 for pkg in pkgs_profile:
                     assert pkg in pkgs_found, '"{}" not in the repository.'.format(pkg)

